config/400-config-istio.yaml

```diff
@@ -41,13 +41,40 @@ data:
     # this example block and unindented to be in the data block
     # to actually change the configuration.
 
+
+    # Documentation below explicits both new and old format.
```

Comment:
Can you integrate that with the docs on the keys? I'm not sure if anybody reads these lines (people tend to look at the keys directly).